test/Microsoft.AspNet.FileProviders.Combined.Tests/TestUtility/MockChangeToken.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.Extensions.Primitives;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.FileProviders.Combined.Tests.TestUtility
+{
+    internal class MockChangeToken : IChangeToken
```

Comment:
I have named it as is to be consistent with the naming in the FileSystemGlobbing test project. But I can change it :smile: